cornice/service.py

```diff
@@ -413,17 +413,21 @@ def cors_enabled(self):
     def cors_enabled(self, value):
         self._cors_enabled = value
 
-    @property
-    def cors_supported_headers(self):
+    def cors_supported_headers(self, method=None):
```

Comment:
When exposing functions with the first argument `method`, we've been following the `{methodname}_for` paradigm, maybe we should stick with this.